astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit fedd7c52ba80e2246f83b8d1752baa3a153a33e2
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
Should not have this in here, and this pops up a couple of times in PRs.  How are you committing changed files in git?  I used to do git commit -am which commits all changed files.  This is risky, as you can end up committing changed files that you may not want to commit.  Several times I ended up committing the astropy_helpers change here when I should not have.